lib/fifo.js

```diff
@@ -565,7 +579,11 @@ class fifo {
         options.callerid = nextcall._options.callerid
       }
 
+      options.late = true
+      options.noAck = true
```

Comment:
Is this needed - this is a drachtio thing - which I intended to be triggered by the late flag?